plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -770,6 +770,109 @@ static void statechart_enact_main_region_Check(Statechart* handle)
 }
 
 
+h2(#cdom_annotations_short_identifiers). The @ShortIdentifiers annotation
+
+It is possible to add annotations to a statechart's specification to alter their or the code generator's behavior, see "Statechart Annotations":../user-guide/statechart_language.textile#sclang_annotations.
+
+The C Domain offers one more annotation: @ShortIdentifiers helps you to keep the generated code compliant to rules which require identifiers to be shorter than 31 characters.
```

Comment:
* Isn't 'shorter than 32 characters' right ? Alternatively : 'require resulting C identifiers to have a maximum length of 31 characters'. 
* We should make clear that we talk about short identifiers in the generated C code so 'C identifiers'.

Reply:
Yes to both.